deny.toml

```diff
@@ -1,6 +1,6 @@
 [licenses]
 version = 2
-allow = ["Apache-2.0", "MIT", "MIT-0", "Unicode-DFS-2016"]
+allow = ["Apache-2.0", "MIT", "MIT-0", "Unicode-3.0"]
```

Comment:
Yeah, the license was changed "[about 5 hours ago](https://crates.io/crates/unicode-ident/versions)" :)

Reply:
Lucky me. :')